doc/why.rst

```diff
@@ -19,33 +19,32 @@ Data-flow programming for performance
   purely functional programming).
 
 Provenance tracking for understanding the code
-...............................................
```

Comment:
This title should be changed in "Provenance tracking to understand the code". My level of English got better in 10 years.